mobile-app/app/screens/AppNavigator/screens/Dex/hook/DexStabilization.ts

```diff
@@ -236,6 +257,22 @@ export function useDexStabilization (tokenA: DexStabilizationTokenA, tokenB: Dex
       }
     }
 
+    // check for direct return path fee info
+    if (
+      bestPath.length === 1 && (
+        (tokenA.displaySymbol === 'DFI' && tokenB.displaySymbol === 'DUSD') ||
```

Comment:
is the swap tokenA to tokenB or tokenB to tokenA. Just from reading it would be tokenB to tokenA. Only if you swap dUSD into DFI, dUSDT or dUSDC the stab.fee is payed.

Reply:
this is only for the return path fee warning message 😉